hudi-common/src/main/java/org/apache/hudi/internal/schema/Type.java

```diff
@@ -62,7 +62,9 @@ enum TypeID {
     DATE(Integer.class),
     BOOLEAN(Boolean.class),
     TIME(Long.class),
+    TIME_MILLIS(Integer.class),
```

Comment:
Should the primitive type for the logical type of time-millis also use `Long` type?